packages/contracts-bedrock/deployments/goerli/OptimismPortal.json

```diff
@@ -523,10 +512,23 @@
     },
     {
       "inputs": [],
-      "name": "systemConfig",
+      "name": "spacer_53_0_1",
```

Comment:
Based on this I'm assuming the deployment is out of date?

Reply:
ah so we need to redeploy optimism portals rip